packages/transforms/snapshot/package.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "@graphql-mesh/snapshot",
```

Comment:
we should change all transforms to be `@graphql-mesh/XXX-transform`, right? @ardatan

Reply:
I only used `transform-xxx` prefix only for conflicted ones. But we can follow this pattern for all our packages?